.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +20,7 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 
 <!--
 Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR work correctly.
+If the changes in your PRs are autogenerated via a script or you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}".
```

Comment:
```suggestion
If the changes in your PRs are autogenerated via a script and you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}".
```

Reply:
@DubeySandeep Like this?